### ISSUE TITLE
Generated constraint does not handle int64 literal properly

### ISSUE THREAD

**Issue by ArcturusZhang** [Azure/autorest.go#181]
The go sdk [code](https://github.com/Azure/azure-sdk-for-go/blob/0820c0553d6c410603056869f7b0e52d61ce5bee/services/netapp/mgmt/2019-08-01/netapp/pools.go#L72) generated from [swagger](https://github.com/Azure/azure-rest-api-specs/blob/96642d3ea33505a43eba61eb7b91e51f11a3a215/specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-05-01/netapp.json#L1395) is not properly handling int64 literal.

```
Chain: []validation.Constraint{{Target: "body.PoolProperties.Size", Name: validation.InclusiveMaximum, Rule: int64(549755813888000), Chain: nil},
    {Target: "body.PoolProperties.Size", Name: validation.InclusiveMinimum, Rule: 4398046511104, Chain: nil},
}
```
In this snippet, number `549755813888000` and `4398046511104` both overflow int32, and are both int64 literal, therefore the number `4398046511104` should also be an int64 literal instead of int, to avoid compilation error on 32-bit machines.

**Comment by ArcturusZhang** [comment]
Related spec pr: https://github.com/Azure/azure-rest-api-specs/pull/7847

**Comment by jhendrixMSFT** [comment]
Fixed in generator version 2.1.138.
